virtme/guest/virtme-init

```diff
@@ -146,7 +147,8 @@ log "basic initialization done"
 ######## The remainder of this script is a very simple init (PID 1) ########
 
 # Does the system use systemd-tmpfiles?
-tmpfiles=`which systemd-tmpfiles 2>/dev/null` && {
+# shellcheck disable=SC2034  # Unused variable left for readability
+tmpfiles=$(command -v systemd-tmpfiles 2>/dev/null) && {
```

Comment:
is it not clearer without the variable? :)

    command -v systemd-tmpfiles 2>/dev/null && { (...) }

Reply:
Actually, this was fixed automatically.

I would rather go with:
```suggestion
if command -v systemd-tmpfiles &>/dev/null; then
```